tools/stub_generator/cuda.json

```diff
@@ -0,0 +1,99 @@
+{
+   "extra_include":[
+      "<string>",
+      "<cuda.h>"
+   ],
+   "return_type":"CUresult",
+   "not_found_error":"CUDA_ERROR_SHARED_OBJECT_SYMBOL_NOT_FOUND",
+   "functions": {
+      "cuInit": {},
+      "cuGetErrorName": {},
+      "cuGetErrorString": {},
+      "cuDriverGetVersion": {},
+      "cuDeviceGetCount": {},
```

Comment:
Just a question: Can this be somehow generated (offline is fine) by listing the symbols from the library? I am wondering if we need to manually add any new symbol that we'd like to use

Reply:
That's something I've already suggested - to have a regex pattern. But maybe let's merge it as-is and improve later - at least more people will have an opportunity to do that when they have the code in their working copies.

Reply:
We can try to regexp inside `stub_codegen.py`. But we can think about this in the next step, now it provides the same functionality as we used to have - hardcoded list of supported functions.